scripts/integration-helpers/framework.ts

```diff
@@ -336,6 +342,10 @@ export class TriggerEndToEndTest extends EmulatorEndToEndTest {
     });
   }
 
+  triggerHttpsFunction(): Promise<Response> {
+    return this.invokeHttpFunction("triggerHttpsFunction");
+  }
+
```

Comment:
hmm I'm not sure I understand this indirection

Why can't we call `invokeHttpFunction` directly in `test.ts` file, similar to how its done here:

https://github.com/firebase/firebase-tools/blob/master/scripts/triggers-end-to-end-tests/tests.ts#L415

Instead of checking to see if a log was emitted, we can rely on the HTTP response to indicate that the request succeeded.

Reply:
Much cleaner, thanks!